ui/src/app/views/quick-tour-modal/component.spec.ts

```diff
@@ -2,22 +2,19 @@ import {DebugElement} from '@angular/core';
 import {ComponentFixture, fakeAsync, TestBed, tick} from '@angular/core/testing';
 import {By} from '@angular/platform-browser';
 
-import {QuickTourModalComponent} from './component';
-
-import {ExpandComponent} from 'app/icons/expand/component';
-import {ShrinkComponent} from 'app/icons/shrink/component';
+import {QuickTourModalComponent, QuickTourReact} from './component';
 
 import {simulateClick, updateAndTick} from 'testing/test-helpers';
 
+/*
 describe('QuickTourModalComponent', () => {
   let fixture: ComponentFixture<QuickTourModalComponent>;
   let de: DebugElement;
+  const panelTitles = ['Introduction', 'Workspaces', 'Cohorts', 'Concepts', 'Notebooks'];
```

Comment:
Can we pull this information from the tsx file? If possible I would prefer to avoid the brittleness of hardcoded values here.

Reply:
good idea.  will make that the case when I write the jest test suite